smoketest/scripts/cli/test_service_ssh.py

```diff
@@ -213,5 +213,50 @@ def ssh_send_cmd(command, username, password, host='localhost'):
         usernames = [x[0] for x in getpwall()]
         self.assertNotIn(test_user, usernames)
 
+    def test_ssh_dynamic_protection(self):
+        """
+        check sshguard service
+        """
+        SSHGUARD_CONFIG = '/etc/sshguard/sshguard.conf'
+        SSHGUARD_PROCESS = 'sshguard'
+        block_time = '123'
+        detect_time = '1804'
+        port = '22'
+        threshold = '10'
+
+        self.cli_set(base_path + ['dynamic-protection', 'block-time', block_time])
+        self.cli_set(base_path + ['dynamic-protection', 'detect-time', detect_time])
+        self.cli_set(base_path + ['dynamic-protection', 'threshold', threshold])
+
+        # commit changes
+        self.cli_commit()
+
+        # Check configured port
+        tmp = get_config_value('Port')
+        self.assertIn(port, tmp)
+
+        # Check sshgurad service
+        self.assertTrue(process_named_running(SSHGUARD_PROCESS))
+
+        sshguard_lines = [
+            'THRESHOLD=10',
+            'BLOCK_TIME=123',
```

Comment:
Please reference to the `block_time` variable above.

`f'BLOCK_TIME={block_time}',`

Reply:
Done